### ISSUE TITLE
When using two 3D scatter plots, if on has selection from grab, scrolling over that plot erases all selections on other plot.

### ISSUE THREAD

**Issue by jmelott** [MD-Anderson-Bioinformatics/ScatterPlotPlugin3D#31]

**Comment by marohrdanz** [comment]
After discussion today, it appears this issue may have resolved itself somewhere along the qa_initial_release branch.

**Comment by marohrdanz** [comment]
Maybe part of this issue is that if you mousedown on the other plot, it clears all the selections (because it's the start of a drag...even if that drag is of nothing).

**Comment by jmelott** [comment]
A drag would imply having the mouse button down.  This happened with no buttons pressed.  It also didn't occur if none of the points on either of the plots had been defined by the grab functionality.

**Comment by jmelott** [comment]
Additional, more detailed information.
Just scrolling over the other plot doesn't cause the issue now, only if you actually hover over a point on the plot that has grabbed portions.  It not only dehighlights the points on one of the maps, it actually gets rid of the selection on the axis in the NGCHM detail for that axis.

To replicate:
Open Deprez3000 map.
Add extra panels such that you have 4 panels.   Not sure if placement is important, but I will list what I used in my test which was a two x two grid of panels.
In the right panels I put the Detail map at the top and the summary map at the bottom.
In the top left panel, I opened a 3D scatterplot with the default column options.
In the lower left panel, I opened a 3D Scatterplot and switched to row.  For each of the 3 coordinates and the color by, I selected various sets of different labels on the detail map and used the grab functionality to define the coordinates.
I selected some rows and columns on the detail map by clicking on some dendrogram regions.  This resulted in a square section of the detail map being selected.  This caused points to be selected on both 3D scatterplot maps as expected.
If I scroll over a point in the upper 3D scatterplot, things seem to work as expected and you see an additional column selected on the detail map temporarily and nothing changes on the lower 3D scatterplot.
If I scroll over a point on the lower 3D scatterplot, the selection on the detail map goes from being a box to being a block of rows that expand across the whole detail map and all of the points in the upper 3D scatterplot are deselected accordingly.  Moving off the point in the lower 3D scatterplot does not reset the selection to the box that was once selected.
